data-collection-scripts/ltc-scraper/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
why its own here vs the repo-wide license? (Ditto on the .gitignore contents, could add to the repo-wide one)